Annotation/Admin.php

```diff
@@ -123,6 +123,11 @@ public function processMetadata(ClassMetadata $metadata)
         $tag = array_filter($tag, function ($v) {
             return !is_null($v);
         });
+
+        if (!empty($this->id)) {
+            $metadata->id = $this->id;
+        }
```

Comment:
the id set's the service id, else it would be autogenerated.

and in the initial implementation I forgot to add it